auth/user_mgt.go

```diff
@@ -1075,17 +1115,27 @@ func (r *userQueryResponse) makeExportedUserRecord() (*ExportedUserRecord, error
 			enrollmentTimestamp = t.Unix() * 1000
 		}
 
-		if factor.PhoneInfo == "" {
+		if factor.PhoneInfo != "" {
+			enrolledFactors = append(enrolledFactors, &MultiFactorInfo{
+				UID:                 factor.MFAEnrollmentID,
+				DisplayName:         factor.DisplayName,
+				EnrollmentTimestamp: enrollmentTimestamp,
+				FactorID:            phoneMultiFactorID,
```

Comment:
Should we still set `PhoneNumber` here? Otherwise the results from `GetUsers()` could be a breaking change, right?

Reply:
Yes, I was slightly confused about whether or not it would seriously affect developers. Thanks for clarifying!